File: Haskell/beginner/1957.hs
-- https://judge.beecrowd.com/pt/problems/view/1957
-- Converter para Hexadecimal

-- Importing Printf to use UpperCase
import Text.Printf (printf)

-- Function to Show Hex in UpperCase
intTextToHexUpper :: Int -> String
intTextToHexUpper = printf "%X"

-- Main
main :: IO ()
main = do
    usrInput <- readLn :: IO Int
    let hexString = intTextToHexUpper usrInput
    putStrLn hexString

-- By: Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1957.c
// https://judge.beecrowd.com/pt/problems/view/1957
// Converter para HexaDecimal

// Libraries
#include <stdio.h>

// Main
int main (){
    int usrInput;
    char usrBuff[20];

    scanf("%d", &usrInput);

    sprintf(usrBuff, "%X", usrInput);

    printf("%s\n", usrBuff);

    return 0;
}

// By: Fernando Serra
// https://github.com/fernandosserra